src/response.rs

```diff
@@ -89,7 +89,7 @@ pub enum Response {
     // ===== Client (Owner) to SrcElders =====
     //
     /// Get a list of authorised keys and the version of the auth keys container from Elders.
-    ListAuthKeysAndVersion(Result<(BTreeMap<PublicKey, AppPermissions>, u64)>),
+    ListAuthKeysAndVersion(Result<(BTreeMap<PublicKey, [u8; 32]>, u64)>),
```

Comment:
Should we be renaming the struct here?

Reply:
Yup! Will make the change along with the changes from the above comment.

Reply:
Refer above comment.